src/java.base/unix/classes/sun/nio/fs/UnixCopyFile.java

```diff
@@ -628,7 +661,10 @@ static void copy(final UnixPath source,
 
     // -- native methods --
 
-    static native void transfer(int dst, int src, long addressToPollForCancel)
+    static native long transferSize0();
```

Comment:
If the `transferSize` is fixed, there seems to be little reason to call native to retrieve it.
(Perhaps hoping for the OS to provide it.)

Reply:
I realized that myself and was already changing it when this comment appeared. Fixed in f151dc4cf6f4e59fff2d2a0f486929940cafb1ae.